packages/frontmatter/package.json

```diff
@@ -22,8 +22,10 @@
     "@headlessui/react": "^1.7.13",
     "@heroicons/react": "^2.0.13",
     "@scienceicons/react": "^0.0.2",
+    "@myst-theme/jupyter": "^0.1.35",
     "classnames": "^2.3.2",
     "myst-common": "*",
+    "thebe-react": "^0.0.7",
```

Comment:
These should be removed from the frontmatter package. I don't think they are actually used anyways!

Reply:
Fixed.